tests/test_convert_lit_checkpoint.py

```diff
@@ -191,3 +191,72 @@ def test_maybe_unwrap_state_dict(tmp_path):
     with mock.patch("scripts.convert_lit_checkpoint.maybe_unwrap_state_dict") as maybe_unwrap:
         convert_lit_checkpoint(checkpoint_name=ckpt_name, checkpoint_dir=tmp_path, model_name=model_name)
     maybe_unwrap.assert_called()
+
+
+def test_maybe_raise_finetune_warning_lora(tmp_path):
+    from lit_gpt.lora import GPT, Config
+    from finetune.lora import save_lora_checkpoint
+    from scripts.convert_lit_checkpoint import convert_lit_checkpoint
+
+    # fabric is needed for finetune.full::save_checkpoint
+    fabric = L.Fabric(devices=1)
+
+    model_name = "Llama-2-7b-hf"
+    ours_config = Config.from_name(model_name, block_size=8, n_layer=2, n_embd=32, n_head=2, padding_multiple=128)
+    ours_model = GPT(ours_config)
+    ours_model.lora_A = torch.nn.Parameter()
```

Comment:
Why do you need this? You can enable lora by setting `r` in the config

Reply:
implemented in [d16241b](https://github.com/Lightning-AI/lit-gpt/pull/397/commits/d16241bda4a44da3135a3f63d7b3c9f116a08494) by setting `r` and `to_mlp`